src/components/App.jsx

```diff
@@ -23,10 +23,12 @@ class App extends React.Component {
     this.getProducts(); // dev tool only
   }
 
-  getProductById(id) {
+  getProductById(event, id) {
```

Comment:
This change would break any other calls to getProductbyId. Make event the second parameter so that other users of this method don't have to refactor

Reply:
Just fixed. Good catch, this was a stupid mistake on my part.